FF1Lib/FF1Rom.cs

```diff
@@ -1183,7 +1187,10 @@ public void Randomize(Blob seed, Flags flags, Preferences preferences)
 
 			if (flags.TournamentSafe || preferences.CropScreen) ActivateCropScreen();
 
-			WriteSeedAndFlags(seed.ToHex(), Flags.EncodeFlagsText(flags));
+			String seedHex = seed.ToHex();
+			String encodedFlagText = Flags.EncodeFlagsText(flags);
+			WriteSeedAndFlags(seedHex, encodedFlagText);
+			StampRom(seedHex, encodedFlagText, FFRVersion.Sha,inputRomHash);
```

Comment:
Do you need these parameters?

If you do this inside WriteSeedAndFlagsAndDigest I think FFRVersion.Sha and inputRomHash are already in scope. Don't need anything new here at all.

Reply:
Im not sure I agree with this one, the WriteSeedsAndFlags does a lot of processing, I think adding this here might obscure what that method is already doing